packages/form-js-editor/src/features/properties-panel/entries/factories/simpleBoolEntryFactory.js

```diff
@@ -8,6 +8,8 @@ export default function simpleBoolEntryFactory(options) {
     description,
     path,
     props,
+    getValue,
```

Comment:
Might have to start renaming it to just boolEntryFactory, but I'm not against the change, less boilerplate is always good.